js/compressed-token/src/program.ts

```diff
@@ -577,8 +581,16 @@ export class CompressedTokenProgram {
     static async createMint(
         params: CreateMintParams,
     ): Promise<TransactionInstruction[]> {
-        const { mint, authority, feePayer, rentExemptBalance, tokenProgramId } =
-            params;
+        const {
+            mint,
+            authority,
+            feePayer,
+            rentExemptBalance,
+            tokenProgramId,
+            freezeAuthority,
+            mintSize,
```

Comment:
adding mint size like this leads to a weird devx.
To initialize token22 mints with extension the instruction order has to be `[createAccount, ...initExtensions, initMint, createTokenPoolIx]`. the current order is `[createMintAccountInstruction, initializeMintInstruction, createTokenPoolIx]` -> one would need to insert init token extension instructions between index 0 and 1 to use the function output to initialize a mint with token22 extensions.

Reply:
We can do it in a different pr but ideally we should pass in the extension types and calculate space and create instructions inside of this function. In the mean time we could adjust the doc string so that it is possible to know where to inject create token 22 init extension instructions.
```
 /**
     * Construct createMint instructions for compressed tokens [createMintAccount, initializeMint, createTokenPool]
     */
 ```

Reply:
added to docstring for now